docker-compose.yml

```
@@ -321,6 +327,17 @@ services:
       - "/config/admin_user_cred.json"
     << : *log-config
 
+  state_groups_cleaner:
+    << : *IMAGE_STATE_GROUPS_CLEANER
+    restart: always
+    volumes:
+      - ./data/db:/data
```

Ah sorry missed this earlier:
```suggestion
      - ${DATA_DIR:-./data}/db:/data
```